### PR TITLE
Adding apns

### PR DESCRIPTION
**Pull request by ccfz**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -14,6 +15,12 @@
 
   get 'match_conversations', to: 'conversations#match_conversation' 
   
+
+  resources :devices, :except => :update
+  scope "devices" do
+    put '', to: 'devices#update'
+  end
+
```
Comment (mwallba) [inline config/routes.rb]: hmm - maybe add some info why you are doing this.
does you client have some kind of strange calling behaviour? do you want to use PUT for idempotency + client is generating UUID's? 
also, to communicate the intention a bit more:
```
resources :devices, except: %i(update) do
  collection do
    put '', to: 'devices#update'
  end
end
```

File: config/application.rb [hunk 2]
```diff
@@ -13,8 +13,11 @@
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
+require 'houston'
```
Comment (mwallba) [inline config/application.rb]: the next line does exactly this for you, unless you put `gem 'houston', require: false` in the Gemfile

File: config/routes.rb (hunk 1 above)
Comment (ccfz) [inline config/routes.rb]: as far as i could tell there was no way to update device properties like age gender etc. what does %i do?

File: config/application.rb (hunk 2 above)
Comment (ccfz) [inline config/application.rb]: ah left over from the billion experiments -.-

**Pull request merged**